style.css

```diff
@@ -0,0 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+body {
+  width: 375px;
```

Comment:
[Optional]

- [ ] It's better if the width is 100%, that way the website will look great for all screen sizes, for example if the screen is slightly bigger than `375px` the page will look cropped like this 👇 

![image](https://user-images.githubusercontent.com/50721479/212069657-d833d7d8-0717-4010-a8bf-9f183fcb9537.png)